testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/server/http/TestHttpServer.groovy

```diff
@@ -249,6 +249,10 @@ class TestHttpServer implements AutoCloseable {
         isTestServer = Boolean.parseBoolean(request.getHeader("is-test-server"))
       }
       if (isTestServer) {
+        def headers = req.getHeaders("traceparent")
```

Comment:
Should we use `GlobalOpenTelemetry.getPropagators().getTextMapPropagator().fields()` instead of hardcoding `traceparent`?

Reply:
oh good idea!